.travis.yml

```diff
@@ -17,4 +17,4 @@ script:
 - ./gradlew ciBuild
 after_success:
 - ./gradlew release
-- ./gradlew cobertura coveralls
+- ./gradlew --debug --stacktrace cobertura coveralls
```

Comment:
Yeah but at that time build/coverage broke without notice just for fun.
I had no clue what was the issue. gradle (and plugin) wasn't much helping on that front. Anyway I removed coveralls and cobertura.